.github/workflows/ci.yml

```diff
@@ -41,9 +41,9 @@ jobs:
         with:
           path: ${{ env.GOPATH }}/src/github.com/karmada-io/karmada
       - name: install Go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v5
         with:
-          go-version: 1.21.8
+          go-version-file: ${{ env.GOPATH }}/src/github.com/karmada-io/karmada/.go-version
```

Comment:
Can you remind me why we have to specify the whole path of `.go-version` here?

Reply:
because the checkout is in a different directory